tests/unit_tests/utils/test_loggings.py

```
@@ -1,11 +1,23 @@
 
-logger = get_logger(__name__)
+@pytest.fixture(scope="module")
+def setup_logging(tmp_path_factory):
+    """Configure logging to use tmp_path for isolated testing"""
+    tmp_dir = tmp_path_factory.mktemp("logs")
+    configure_logging(debug=True, log_dir=str(tmp_dir))
+    return tmp_dir
+
+
+@pytest.fixture(scope="module")
+def logger(setup_logging):
```

Ruff (0.15.0)

[warning] Unused function argument: `setup_logging`

(ARG001)